plugins/modules/vmware_guest_disk.py

```diff
@@ -709,7 +770,7 @@ def sanitize_disk_inputs(self):
                                 autoselect_datastore=True,
                                 disk_unit_number=0,
                                 controller_number=0,
-                                disk_mode='persistent',
+                                disk_mode='independent_persistent',
```

Comment:
The default disk mode is "persistent" when adding a new disk. Why here to change to "independent_persistent"?

Reply:
@Tomorrow9 looks like I missed this while I as testing. I have reverted it back to 'persistent'